### PR TITLE
chore: add some link to rustdoc

### PR DESCRIPTION
**Pull request by linyihai**
Mention the rustdoc in README,  this could be useful for who curious about the documentation rendered.

Fixes https://github.com/rust-lang/docs.rs/issues/2220

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,6 +11,12 @@ the nightly release of the Rust compiler.
 
 This readme is for developing docs.rs. See [the about page](https://docs.rs/about) for user-facing documentation.
 
+
+## How has the rust doucumentation been rendered
```
Comment (syphar) [inline README.md:15]:
```suggestion
## How the documentation is generated 
```

File: README.md [hunk 2]
```diff
@@ -11,6 +11,12 @@ the nightly release of the Rust compiler.
...
+
+rustdoc generates documentation for Rust project, its respository in [here](https://github.com/rust-lang/rust/tree/master/src/librustdoc).
```
Comment (syphar) [inline README.md:17]: docs.rs uses [rustdoc](https://github.com/rust-lang/rust/tree/master/src/librustdoc) to generate the documentation for every crate release on crates.io.

File: README.md [hunk 3]
```diff
@@ -11,6 +11,12 @@ the nightly release of the Rust compiler.
...
+You can read the [The rustdoc book](https://doc.rust-lang.org/nightly/rustdoc/what-is-rustdoc.html) for more details.
```
Comment (syphar) [inline README.md:18]:
```suggestion
You can read the [the rustdoc book](https://doc.rust-lang.org/nightly/rustdoc/what-is-rustdoc.html) for more details.
```

**Review by syphar — CHANGES_REQUESTED** [review]
Thank you for the help!

File: README.md (hunk 2 above)
Comment (linyihai) [reply README.md]: I had adopt your advice already.

**Review by syphar — APPROVED** [review]

**Pull request merged**